src/cli/mod.rs

```diff
@@ -66,7 +67,7 @@ pub enum CargoMsrvCli {
     /// Find your Minimum Supported Rust Version!
     #[command(
         author = "Martijn Gribnau <garm@ilumeo.com>",
-        after_help = r#"
+        after_help = indoc::indoc!{"
```

Comment:
Does this actually fix anything or is it just a nice to have (i.e. is it worth to add a dependency for this)?

Reply:
It unindents the text. Makes it look less weirdly wrapped. The commit is separate if you only want the colors.